community.json

```diff
@@ -639,6 +639,15 @@
       "tags": ["ca", "crow", "sequencer"],
       "origin": "lines"
     },
+    {
+      "project_name": "less_concepts_3",
+      "project_url": "https://github.com/dndrks/less_concepts_3",
```

Comment:
woop! @linusschrab , this should be https://github.com/linusschrab/less_concepts_3